java/google/registry/rde/imports/RdeImportUtils.java

```diff
@@ -66,10 +83,11 @@ public RdeImportUtils(
     this.escrowBucketName = escrowBucketName;
   }
 
```

Comment:
true. That would introduce more changes that are unrelated to the scope of this PR, let me know if you feel strongly about it :)